### ISSUE TITLE
Review warnings from OBO dashboard ROBOT report

### ISSUE THREAD

**Issue by sbello** [mgijax/mammalian-phenotype-ontology#3905]
See https://dashboard.obofoundry.org/dashboard/mp/robot_report.html
duplicate synonyms - 7 terms
missing definitions 119 terms

**Comment by sbello** [comment]
MP:0009142 and MP:0014256; MP:0008244and MP:0030944 problem is that the 2 terms share an acronym so the same synonym is an exact match to both - we should look into using the acronym tag but need to consider what this would do to the MP loads at MGI and other resources. Related to this is the use of the acronym ASD in MP and HPO
[Autistic behavior](https://www.informatics.jax.org/vocab/hp_ontology/HP:0000729) (ASD)
[Atrial septal defect](https://www.informatics.jax.org/vocab/hp_ontology/HP:0001631) (ASD)
[atrial septal defect](https://www.informatics.jax.org/vocab/mp_ontology/MP:0010403) (ASD)
[ostium primum atrial septal defect](https://www.informatics.jax.org/vocab/mp_ontology/MP:0010404) (ostium primum ASD)
[ostium secundum atrial septal defect](https://www.informatics.jax.org/vocab/mp_ontology/MP:0010405) (ostium secundum ASD)
[common atrium](https://www.informatics.jax.org/vocab/mp_ontology/MP:0010406) (common atrium ASD)
[coronary sinus atrial septal defect](https://www.informatics.jax.org/vocab/mp_ontology/MP:0010407) (coronary sinus ASD)
[sinus venosus atrial septal defect](https://www.informatics.jax.org/vocab/mp_ontology/MP:0010408) (sinus venosum ASD)

Results in incorrect lexical mapping between atrial septal defect in MP and autistic disorder in HPO

MP:0014312 should have been thick not thin
MP:0010575 report has 'duplicate label synonym' but the label is 'aortic arch dilation' and the synonym is 'aorta arch dilation' these are not duplicates

**Comment by sbello** [comment]
[MP:0011510](http://purl.obolibrary.org/obo/MP_0011510) - requested PMID:23804932 to try to define

**Comment by sbello** [comment]
problem terms

- intraembryonal intestine elongation - only annotations are from DMDD - maybe this was meant to be abnormal intestinal elongation or increased or???
- no idea what the proximal internal carotid artery is, I find references to this primarily related to proximal obstruction one reference wrote this as proximal (extracranial) ICA but no clue if this is the common way to refer to this
-
